LFPy/recextelectrode.py

```diff
@@ -40,6 +40,9 @@ class RecExtElectrodeSetup(object):
         n : int,
             if N is not None and r > 0, the number of points to use for each
                   contact point in order to calculate average
+        square : bool,
+            if square is True, the electrode shape is a square and r is used
+                  as square side
```

Comment:
@alejoe91 This description is not in sync with the new input kwarg "shape", please revise. Otherwise it looks fine.